finality-aleph/src/data_io.rs

```diff
@@ -25,8 +25,13 @@ const AVAILABLE_BLOCKS_CACHE_SIZE: usize = 1000;
 const MESSAGE_ID_BOUNDARY: MessageId = 100_000;
 const PERIODIC_MAINTENANCE_INTERVAL: Duration = Duration::from_millis(60000);
 
+pub(crate) type AlephData<B> = (
```

Comment:
Slight preference – the references to `block_data.[10]` are kinda ugly, it would be nicer to have more descriptive names. At this point with just two fields this is readable though, so I don't consider this a blocker.

Reply:
I implemented AlephData as a struct, but because of it being generic dependent on `Block` trait I had to manually implement `Hash` trait, which lead to manually implementing `PartialEq` and `Eq`.

Reply:
I think that's alright, although see my idea on how to potentially fix this in the other comment thread.